src/Microsoft.Android.Build.BaseTasks/Microsoft.Android.Build.BaseTasks.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\tools\BuildTasks.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Microsoft.Android.Build.Tasks</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <GenerateResxSource>true</GenerateResxSource>
+    <UpdateXlfOnBuild Condition=" '$(RunningOnCI)' != 'true' ">true</UpdateXlfOnBuild>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" Version="16.5.0" IncludeAssets="compile" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="16.5.0" IncludeAssets="compile" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.5.0" IncludeAssets="compile" />
+    <PackageReference Include="Xamarin.Build.AsyncTask" Version="0.3.4" IncludeAssets="compile" />
+    <PackageReference Include="Xamarin.LibZipSharp" Version="1.0.20" IncludeAssets="compile" />
```

Comment:
@jonathanpeppers  /  @dellis1972  do you think it makes sense to try to better sync these versions with xamarin-android and monodroid? Maybe we should move part of https://github.com/xamarin/xamarin-android/blob/master/build-tools/scripts/MSBuildReferences.projitems here?

Reply:
I think moving these to their own MSBuild file makes sense. They could be imported from other repos.

The only problem I see is the libZipSharp version. Is it OK to move the version to this repo? libZipSharp bumps would go from xamarin-android-tools then to xamarin-android?

The libZipSharp version is currently here:

https://github.com/xamarin/xamarin-android/blob/087c7f43a1a60a6c92934d0a57c902e381877df3/Configuration.props#L127

Reply:
Yeah I'm wondering if this will introduce more of a headache than the alternative -- keeping versions in multiple repos in sync...

The biggest issue I see with LibZipSharp is: https://github.com/xamarin/xamarin-android/blob/bea063b939d7d8d5f166c50564665d62ab140417/build-tools/xaprepare/xaprepare/ThirdPartyNotices/LibZipSharp.cs#L16, but I suppose we could update that to read from a file in this repo instead.

I'll move the version properties into a separate file that can be referenced elsewhere for now, and we can decide if we want to use it everywhere later.

Reply:
We have this file https://github.com/xamarin/xamarin-android-tools/blob/main/src/Xamarin.Android.Tools.AndroidSdk/Xamarin.Android.Tools.Versions.props which is imported in xa 
so perhaps the LibZipSharp versions can go in there?

Reply:
although its not imported as part of the build only in Xamarin.Android.Common.props.

Reply:
I think we ship the `Xamarin.Android.Tools.Versions.props` file in our installers as well, so it may be preferable to add a new file for these product build dependency related items?